config/jobs/kubernetes/sig-cluster-lifecycle/kubespray.yaml

```diff
@@ -0,0 +1,15 @@
+# kubeadm release and master tests
+periodics:
+- name: ci-kubernetes-e2e-kubeadm-kubespray-gce-master
+  interval: 2h
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/e2e-kubeadm:v20181024-9b3b78abc
+      args:
+      - --repo=k8s.io/kubernetes=master
+      - --repo=github.com/kubernetes-incubator/kubespray=master
```

Comment:
It's not really clear to me how kubespray takes the periodic built artifacts, provisions a cluster, and runs the tests.

Reply:
just comparing to how k-a works:

the second `--repo` pulls the kubespray repo and from there any `.sh` script from that repo can be called, but i think such calls should be made from a kubetest deployment (plugin), because the plugin
would be able to obtain a GCE project and also integrate with the ginkgo / e2e suite.

not sure what alternatives are available to this.

ideally we should have a separate call with folks from sig-testing + kubespray to discuss a path forward.